Sources/ArgumentParser/Parsable Types/ParsableCommand.swift

```diff
@@ -164,4 +171,19 @@ extension ParsableCommand {
   internal static var defaultIncludesUnconditionalArguments: Bool {
     configuration.defaultSubcommand?.includesUnconditionalArguments == true
   }
+  
+#if DEBUG
+  @available(macOS 10.15, macCatalyst 13, iOS 13, tvOS 13, watchOS 6, *)
+  internal static func checkAsycHierarchy(_ command: ParsableCommand.Type) {
```

Comment:
Spelling nit on this:
```suggestion
  internal static func checkAsyncHierarchy(_ command: ParsableCommand.Type) {
```

Reply:
Oops!